CHANGELOG.md

```diff
@@ -82,6 +90,26 @@
 
 [storage-mocking-link]: https://googleapis.dev/cpp/google-cloud-storage/latest/storage-mocking.html
 
+**OTHER CHANGES**:
+* feat(storage): discard handles after an error (#7088)
+* feat(storage): release download handles sooner (#7085)
+* feat(storage): support setting HTTP version (#7077)
+* fix(storage): cleanup interrupted downloads (#7064)
+* test(storage): make ScheduleForDelete() thread-safe (#7063)
+* fix(storage): avoid crashes after move (#7045)
+* feat(GCS+gRPC): upload checksums with last chunk (#7031)
+* fix(storage): use hash values in InsertObject() (#7025)
+* feat(storage): capture peer address for REST (#6994)
+* feat(GCS+gRPC): option to configure plugin (#6991)
+* feat(GCS+gRPC): checksums on ObjectInsert (#6967)
+* fix(GCS+gRPC): no hashes in Object resource (#6963)
+* feat(storage): use Base64 transforms (#6956)
```

Comment:
Probably remove this one, it is basically a refactor, customers would not care.

Reply:
Removed